menuinst/platforms/base.py

```diff
@@ -37,7 +37,10 @@ def __init__(
         self.prefix = Path(prefix)
         self.base_prefix = Path(base_prefix)
 
-        self.env_name = None
+        if self.prefix.absolute() == self.base_prefix.absolute():
```

Comment:
I'm not sure this handles symlinks well. See [discussion here](https://discuss.python.org/t/pathlib-absolute-vs-resolve/2573/19) for more details about `.absolute()` being not recommended in general.

Instead use [`.samefile()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.samefile) which [does follow symlinks](https://stackoverflow.com/questions/71939549/does-os-path-samefile-follow-symbolic-links).